kview/src/main/java/dev/ikm/komet/kview/events/AddStampEvent.java

```diff
@@ -0,0 +1,18 @@
+package dev.ikm.komet.kview.events;
+
+import dev.ikm.komet.framework.events.Evt;
+import dev.ikm.komet.framework.events.EvtType;
+import dev.ikm.komet.framework.view.ViewProperties;
+import dev.ikm.tinkar.common.id.PublicId;
+
+public class AddStampEvent extends Evt {
+
+    public static final EvtType<EditOtherNameConceptEvent> ADD_STAMP = new EvtType<>(Evt.ANY, "ADD_STAMP");
```

Comment:
May be able to remove because you can emit an Evt.ANY. If the subscriber doesn't care about the event type. then the publisher can just pass an Evt.ANY.